data/Relationships/One2MBeanRelationship.php

```diff
@@ -263,8 +269,8 @@ public function getQuery($link, $params = array())
                 return $query;
             } else {
                 return array(
-                    'select' => "SELECT {$this->def['rhs_table']}.id",
-                    'from' => "FROM {$this->def['rhs_table']}",
+                    'select' => "SELECT {$rhsTable}.id",
```

Comment:
Could you escape the values being entered into the query using the quote or quoted function.

Reply:
Hi @code-ph0y , I have refactored the code of getQuery function using quote and quoted functions when needed. Do you suggest any more changes to improve this PR?